app/models/state_file_az_intake.rb

```diff
@@ -175,4 +175,39 @@ def filing_status
     return :head_of_household if direct_file_data&.filing_status == 5 # Treat qualifying_widow as hoh
     super
   end
+
+  def requires_hoh_qualifying_person_name?
+    filing_status == :head_of_household
```

Comment:
Also QSS? Not sure what QSS is but saw it in the story 
```
populate this line on XML and PDF for all AZ HOH filers (including those who were federal QSS filers). 
```

Reply:
QSS is Qualifying Surviving Spouse, which is a recent rename from Qualifying Widow. (We still use "Widow" throughout most of our non-user facing code.) AZ defaults QSS to HOH, which was implemented as a previous story. The `StateFileAzIntake` overloads the `filing_status` to return HOH if filing as QSS. I can add a test case for this though so that if this logic changes then it won't break.